ceph/rados/core_workflows.py

```diff
@@ -2877,3 +2877,218 @@ def set_noautoscale_flag(self, retry: int = 15) -> bool:
 
         log.error("Noautoscale flag not set on the cluster. Returning Fail..")
         return False
+
+    def create_rbd_image(self, pool_name, img_name, **kwargs):
```

Comment:
This module is already available in RBD workflows, Is there a need to create a duplicate module in our workflow to achieve the same thing?

Reply:
I have not reviewed the RBD test modules. I feel it's fine. Do you feel that this should be removed?

Reply:
I had the same thought. The basic RBD functions are already exisiting in the repo. Im good with both, keep them or remove them.

Reply:
IMO wee can keep them for now and but should definitely review duplicate modules later, maybe once we are done with tier automation